spec/take_away_spec.rb

```diff
@@ -0,0 +1,106 @@
+require './lib/take_away'
+describe TakeAway do
+  let(:menu) { double :menu, display_menu: nil }
+  let(:dish) { double :dish }
+  let(:dish2) { double :dish2 }
+  let(:price) { double :price }
+  let(:sms_sender) { double :sms_sender, text_confirmation: true }
+  subject(:takeaway) { described_class.new(menu, sms_sender) }
+  describe "#initialize" do
+    it 'creates an empty list of orders' do
+      expect(takeaway.order_list).to eq []
+    end
+    it 'creates a menu' do
+      expect(takeaway.menu).to eq menu
+    end
+  end
+
+  describe '#display_menu' do
+    it 'calls method display menu on menu instance' do
+      expect(takeaway.menu).to respond_to(:display_menu)
+      takeaway.display_menu
```

Comment:
this is very close to the correct test, but the expectation probably shouldn't be for the menu to respond_to display_menu - we know this is true because you've set up the double to respond to it up above.

Instead you should think about using to receive:

```ruby
 it 'calls method display menu on menu instance' do
      expect(takeaway.menu).to receive(:display_menu)
      takeaway.display_menu
end
```